default/stringtables/en.txt

```diff
@@ -1147,6 +1147,18 @@ Gas Giants Habitable size
 RULE_HABITABLE_SIZE_DESC
 Value used to adjust for [[encyclopedia POPULATION_TITLE_SHORT_DESC]] changes on a planet of this size.
 
+RULE_THRESHOLD_HUMAN_PLAYER_WIN
+Limit human player winners
+
+RULE_THRESHOLD_HUMAN_PLAYER_WIN_DESC
+Limit count of alive human players for them to be considered winners together. Not active for single-player.
+
+RULE_ONLY_ALLIANCE_WIN
+Only allianced players could win
+
+RULE_ONLY_ALLIANCE_WIN_DESC
+Allow human players to be winners only if them in alliance with each other. Not active for single-player.
```

Comment:
This is grammatically bad ("them") and also unclear or misleading. Can a player only win if allied, meaning a player can't win if there are no other human players in the game, or does it mean multiple players can simultaneously win only if they are all in a single alliance, or just some of them need to be allied for a mix of allied and non-allied players to win, or they all need to be allied but not necessarily in a single alliance?

Reply:
It should mean this rule should ensure there is no pair of human players in other then alliance diplomacy status. Not sure how better rephrase it.

Reply:
Tried to rephrase it. Should fixed misleading.